esinstall/src/index.ts

```diff
@@ -175,6 +199,8 @@ function resolveWebDependency(
       foundEntrypoint[dep] ||
       foundEntrypoint['./index.js'] ||
       foundEntrypoint['./index'] ||
+      foundEntrypoint['index.js'] ||
+      foundEntrypoint['index'] ||
```

Comment:
If an export map doesn‘t start with `./` for an entry, it should be invalid. Is there a way we can sanitize this so that `index` is `./index` by the time we get here (i.e. if we control `foundEntrypoint`, we should never allow invalid names like `index`)?

Reply:
This is the `browser` map, not the export map.

Reply:
Oh, sorry. Carry on then!